duckplayer/duckplayer-impl/src/main/java/com/duckduckgo/duckplayer/impl/DuckPlayerPixelName.kt

```diff
@@ -33,4 +33,12 @@ enum class DuckPlayerPixelName(override val pixelName: String) : Pixel.PixelName
     DUCK_PLAYER_SETTINGS_PRESSED("duckplayer_setting_pressed"),
     DUCK_PLAYER_NEWTAB_SETTING_ON("duckplayer_newtab_setting-on"),
     DUCK_PLAYER_NEWTAB_SETTING_OFF("duckplayer_newtab_setting-off"),
+    DUCK_PLAYER_YOUTUBE_ERROR_SIGN_IN_REQUIRED_IMPRESSION("duckplayer_youtube-signin-error_impression"),
```

Comment:
Since these pixels are fired from the app module, they should be moved to AppPixelName. Other modules should never access the impl directly